build/org.eclipse.birt.target/org.eclipse.birt.target.target

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?pde version="3.8"?>
+<target includeMode="feature" name="BIRT Photon target platform">
+	<locations>
```

Comment:
I suggest to sort `location` elements according to its `repository.location` attribute value
   and to sort `unit` elements according to its `id` value
That could give clear (and automatable) normalization rule for .target content.